Remove duplication in kitchen and add enable pgdg to 1204.

cmluciano opened the pull request:
PR(#86) set the default postgresql to 9.3 which does not exist
by default in the ubuntu-12.04 repos. I added the enable_pgdg_apt
option by default for 12.04.

I also removed unnecessary attribute overrides in the kitchen file
since the options are set in the default attributes.

legal90 commented on .kitchen.yml:
```diff
@@ -20,15 +20,9 @@ platforms:
-    attributes:
-      postgres:
-        version: "9.3"
-  - name: centos-6.5
```
I think, there is no need to change the name `centos-6.5` -> `opscode-centos-6.5`, isn't it?

cmluciano commented:
@legal90 you're right. I think I had this changed locally or test-kitchen was having trouble pulling in this box. I have reverted this name and fixed the tests.

smith commented:
:+1: thanks!
